zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -118,3 +117,42 @@ def _update_attribute(self, attrid, value):
         (LONG_PRESS, LONG_PRESS): {COMMAND: COMMAND_HOLD},
         (LONG_RELEASE, LONG_RELEASE): {COMMAND: COMMAND_HOLD},
     }
+
+
+class SwitchAQ3B(XiaomiCustomDevice):
+    """Aqara button device - alternate version."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=24321
```

Comment:
The device type is incorrect, it should be 259 (0x103).

Reply:
Cool, I’ll fix and merge. Thx for testing!